sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/eventhubs/AzureEventHubsMessagingAutoConfigurationTests.java

```diff
@@ -65,10 +73,57 @@ void connectionInfoAndCheckpointStoreProvidedShouldConfigure() {
             )
             .withUserConfiguration(AzureEventHubsPropertiesTestConfiguration.class)
             .withBean(CheckpointStore.class, TestCheckpointStore::new)
+            .withBean(ObjectMapper.class)
             .run(context -> {
                 assertThat(context).hasSingleBean(EventHubsProcessorFactory.class);
                 assertThat(context).hasSingleBean(AzureEventHubsMessagingAutoConfiguration.ProcessorContainerConfiguration.class);
             });
     }
 
+    @Test
+    void withoutObjectMapperShouldNotConfigure() {
+        this.contextRunner
+            .withClassLoader(new FilteredClassLoader(ObjectMapper.class))
+            .withPropertyValues(
+                "spring.cloud.azure.eventhubs.connection-string=" + String.format(CONNECTION_STRING_FORMAT, "test-namespace")
+            )
+            .run(context -> assertThat(context).doesNotHaveBean(EventHubsMessageConverter.class));
+    }
+
+    @Test
+    void withObjectMapperShouldConfigure() {
+        this.contextRunner
+            .withPropertyValues(
+                "spring.cloud.azure.eventhubs.connection-string=" + String.format(CONNECTION_STRING_FORMAT, "test-namespace")
+            )
+            .withUserConfiguration(AzureEventHubsPropertiesTestConfiguration.class)
+            .withBean(ObjectMapper.class)
```

Comment:
Do we need to test the ObjectMapper from spring boot autoconfiguration can be configured to ours? then here we shouldn't manually set the `ObjectMapper` to the test context, but add that spring boot jackson autoconfiguartion

Reply:
how about this?